app-frontend/src/app/components/lab/templateItem/templateItem.module.js

```diff
@@ -10,10 +10,39 @@ const TemplateItemComponent = {
 };
 
 class TemplateItemController {
-    constructor() {
+    constructor($rootScope, $log, userService) {
         'ngInject';
+        $rootScope.autoInject(this, arguments);
         this.BUILDCONFIG = BUILDCONFIG;
     }
+
+    $onInit() {
+        this.getTemplateOwner();
+    }
+
+    getTemplateOwner() {
+        if (this.templateData.owner === 'default') {
+            this.templateOwner = this.BUILDCONFIG.APP_NAME;
+        } else {
+            this.userService.getUserById(this.templateData.owner).then(user => {
```

Comment:
Doing this request for every template item makes me a little uneasy but it's unavoidable for now. We should probably make an issue to include user info along with templates and other items that can be shared just as we do for projects.

Reply:
This is a good idea. A new issue for it is here: https://github.com/raster-foundry/raster-foundry/issues/4182